test/fixtures/for/in-auto-const/actual.js

```diff
@@ -1 +1 @@
-for k in {}: k
+for const k in {}: k
```

Comment:
`auto-const` still happens in most other parts of the language, just no longer in `for-in` and `for-of`, so we'll want to retain most of those tests.